Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/VungleAdapter.kt

```diff
@@ -31,6 +32,11 @@ class VungleAdapter(providerName: String) : AbstractAdapter(providerName),
         // The network's capability to load a Rewarded Video ad while another Rewarded Video ad
         // of that network is showing
         mLWSSupportState = LoadWhileShowSupportState.LOAD_WHILE_SHOW_BY_INSTANCE
+
+        // Set network SDK name and version
+        VungleAds.setIntegrationName(
```

Comment:
@ManojBudumuru-vungle Maybe we should do this for iOS too. Move the integration set name to here instead of the init part.